Sming/SmingCore/Network/Http/HttpServerConnection.cpp

```diff
@@ -505,34 +505,20 @@ void HttpServerConnection::onError(err_t err)
 	TcpClient::onError(err);
 }
 
-const char* HttpServerConnection::getStatus(enum http_status code)
```

Comment:
Don't remove `HttpServerConnection::getStatus` because you will break compatibility. Leave it and use`httpGetStatusText` internally.

Reply:
Tricky one that, since getStatus() returns const char*. All we could do there is leave the current implementation as-is and deprecate it. Plus, it should be a static method.

Reply:
I'll leave as-is then.